worksheet_B/README.md

```diff
@@ -0,0 +1,30 @@
+```
+while IncorrectGuesses < 4                                      # loops until the user has 4 inccorect guesses
```

Comment:
Pseudocode looks good 👍 

Minor nitpicks: you should specify that IncorrectGuesses is initialised to 0 and SecretWord is initialised to a randomly selected word.